bc_obps/registration/api/v1/_user_operators/user_operator_id.py

```diff
@@ -15,22 +15,34 @@
 from uuid import UUID
 from service.error_service.custom_codes_4xx import custom_codes_4xx
 
-## POST
-@router.post(
-    "/user-operator/operator",
-    response={200: RequestAccessOut, custom_codes_4xx: Message},
-    url_name="create_operator_and_user_operator",
+from service.data_access_service.user_operator_service import UserOperatorDataAccessService
+
+from registration.schema.v1 import UserOperatorOut
+
+from registration.models import (
+    AppRole,
+)
+
+# GET
+@router.get(
+    # brianna method problems if it's user-operators
+    "user-operator/{uuid:user_operator_id}",
```

Comment:
This should be called `user-operators` not `user-operator` (singular), but if I change it the endpoint tests start failing with 405 errors. Thoughts? I'm still working on it simultaneously as this PR is being reviewed